cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -2144,6 +2165,9 @@ declare namespace Deno {
    * console.log(new TextDecoder().decode(Deno.readFileSync("my_file.txt"))); // H
    * ```
    *
+   * @deprecated (suggestion). {@linkcode Deno.fsyncSync} will be removed in
```

Comment:
What do we want to suggest to the user with these deprecated APIs containing "(suggestion)"?